twake/backend/node/src/services/channels/services/channel/service.ts

```diff
@@ -129,6 +130,12 @@ export class Service implements ChannelService {
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         (channelToSave as any)[field] = channel[field];
       });
+
+      localEventBus.publish<ResourceEventsPayload>("channel:updated", {
+        actor: context.user,
+        resourcesBefore: [channelToUpdate],
+        resourcesAfter: [channel],
+      });
```

Comment:
OK to keep it here, I will find a coller way to do this

Reply:
Should I write a comment about this?

Reply:
No it is OK